worker/main.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"os"
+	"os/signal"
+	"syscall"
+
+	"google-scraper/database"
+	_ "google-scraper/initializers"
+	"google-scraper/worker/jobs"
+
+	"github.com/beego/beego/v2/core/logs"
+	"github.com/gocraft/work"
+	"github.com/beego/beego/v2/server/web"
+)
+
+var crawlingJobName string
+var enqueuerName string
+
+func init() {
+	var err error
```

Comment:
Why do we need to declare the variable here? I think we can just use something like this
```go
crawlingJobName, err := web.AppConfig.String("crawlingJobName")
.
.
.
enqueuerName, err := web.AppConfig.String("redisEnqueuerName")
```

Reply:
To reuse the previous variables(crawlingJobName, enqueuerName), I had to define the error.
![Screen Shot 2021-03-16 at 11 50 23 AM](https://user-images.githubusercontent.com/6076762/111257564-f242cd00-864d-11eb-8870-4b3c6f5119c7.png)